benchmark/cmd/load.go

```
@@ -0,0 +1,331 @@
+	defer stream.Close()
+	
+	// Send some commands
+	commands := []string{"echo 'Load test active'", "date", "pwd"}
+	go func() {
+		for i, cmd := range commands {
+			select {
+			case <-stopChan:
+				return
+			default:
+				time.Sleep(time.Duration(i+1) * time.Second)
+				c.SendInput(sessionID, cmd+"\n")
```

golangci-lint (1.64.8)

Error return value of `c.SendInput` is not checked

(errcheck)